docusaurus/docs/Android/03-ui/04-message-components/04-message-composer.mdx

```diff
@@ -363,7 +366,7 @@ Many attributes of this View can be configured, like changing its color, the bor
 Here's an example of setting a custom attribute:
 
 ```xml
-<io.getstream.chat.android.ui.message.composer.MessageComposerView
+<io.getstream.chat.android.ui.feature.messages.composer.MessageComposerView
```

Comment:
I think we could improve the text above saying:

```
Many attributes of this View can be configured, like changing its color, the border and the color of the message input, fonts, components visibility, and so on. The full list of available attributes can be found [here](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/attrs_message_composer_view.xml).
```

Maybe it could end up being something like this

```
The styling of the View can be configured by styled attributes. You can change the color of the message input, the fonts, visibility of various components  and so on. The full list of available attributes can be found [here](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/attrs_message_composer_view.xml).
```

And the text below that stating:

```
Different configurations can be used to achieve the desired appearance of MessageComposerView. If you don't need to change this View appearance at runtime, XML should be enough. But if you want to customize it at runtime, then you can use MessageComposerViewStyle as described in the next section.
```

Could become

```
Different configurations can be used to achieve the desired appearance of `MessageComposerView`. If you don't need to change the View's appearance at runtime, using styled attributes should be enough. However, if you want to customize it at runtime, then you can use `MessageComposerViewStyle` as described in the next section.
```

Reply:
Note these are suggestions and not 100% proofread so feel free to gauge yourself if you want to adopt them and if something needs to be changed.